components/navigation/navigation.css

```diff
@@ -0,0 +1,122 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: "Roboto", sans-serif;
+}
+
+.ha-content > * {
+  padding: 0.625rem;
+  text-align: center;
+}
+
+/* Header */
+
+.ha-header {
+  grid-area: header;
+  align-items: center;
+  justify-content: center;
+  display: flex;
+  background-color: rgb(235, 231, 231);
```

Comment:
use color variable instead of hard coded value

Reply:
done